packages/sdk/src/modules/did/index.ts

```diff
@@ -148,6 +174,98 @@ export class Did extends Base {
     }
   }
 
+  // TODO create requirements so that the DID Document is updated correctly based on the peaq-did-proto
+  // - read to make sure there is one?
+  // - how to confirm the previous verifications if they add a new one
+  public async update(options: UpdateDidOptions,
+    statusCallback?: (result: ISubmittableResult) => void | Promise<void>
+  ): Promise<UpdateDidResult | null> {
+    try {
+      const api = this._getApi();
+
+      const { name, address = '', seed = '', customDocumentFields } = options;
+
+      const keyPair = this._metadata?.pair || this._getKeyPair(seed);
+      const accountAddress = address || keyPair.address;
+
+      if (!name) throw new Error('Name is required');
+      if (!accountAddress) throw new Error('Address is required');
+      if (!customDocumentFields) throw new Error('DID Document fields must be configured before manually changing.');
+
+      
+      // TODO: read did to get back did document to add new data to
+      // - discuss how to properly update and implement
+      const did_document = await this.read({name, address});
```

Comment:
Reads previous did document that is affiliated with the account at the name. 

TODO -> decide how to update the previously read did document. Function below to do that called _updateDidDocument(), but it has not been implemented. Currently the code simply creates a new DID Document passed on customFields sent.